back/src/common/constants/companySearchHelpers.ts

```diff
@@ -68,12 +71,64 @@ export const countries = [
   switzerland,
   unitedKingdom
 ];
+export const TEST_COMPANY_PREFIX = "000000";
+
+// support all environments front and back
+// process.env is not available in frontend JS
+let ALLOW_TEST_COMPANY = false;
+try {
+  ALLOW_TEST_COMPANY = process.env.ALLOW_TEST_COMPANY === "true";
+} catch (e) {
+  ALLOW_TEST_COMPANY = false;
+}
```

Comment:
nitpick, mais ce genre de truc serait peut être plus simple ?
```suggestion
const ALLOW_TEST_COMPANY = process?.env?.ALLOW_TEST_COMPANY === "true";
```

Reply:
mais ça ne marche pas, ça fait planter le front